admin/controllers/questions.js

```diff
@@ -0,0 +1,48 @@
+const CheckForm = require('../models/check-form')
+const Pupil = require('../models/pupil')
+const School = require('../models/school')
+const { QUESTION_TIME_LIMIT, TIME_BETWEEN_QUESTIONS } = require('../config')
+
+/**
+ * Returns the set of questions, pupil details and school details in json format
+ * @param req
+ * @param res
+ * @returns { object }
+ */
+
+const getQuestions = async (req, res) => {
+  const { pupilId, schoolId } = req.body
+  if (!pupilId || !schoolId) res.status(400).send('invalid input')
+  let checkForm, pupil, school
+  try {
+    // Until we determine the logic behind fetching the appropriate check form
+    // the pupil will receive the first one
+    checkForm = await CheckForm.findOne({}).sort({createdAt: 1}).lean().exec()
+    pupil = await Pupil.findOne({ '_id': pupilId }).lean().exec()
+    school = await School.findOne({_id: schoolId}).lean().exec()
+  } catch (error) {
+    throw new Error(error)
```

Comment:
@GuyHarwood - no the mongoose calls do not match the existing pupil app:

note that there is a requirement in the existing app to store certain login attempt information.  From a reporting perspective I would suggest enhancing it to match the logging done in the admin app rather than removing it.
```
const school = await School.findOne({schoolPin}).exec()

    if (!school) {
      // school not found
      await saveFailedLogin(logonEvent, req, school, null)
      return done(null, false)
    }

    const pupil = await Pupil.findOne({
      school: school._id,
      pin: pupilPin,
      pinExpired: false
    }).exec()
```